polynote-kernel/src/main/scala/polynote/kernel/interpreter/python/PythonInterpreter.scala

```diff
@@ -229,6 +261,10 @@ class PythonInterpreter private[python] (
       |        else:
       |            return node
       |
+      |def __polynote_mkindex__(l1, l2):
+      |    import pandas as pd
```

Comment:
What if `pandas` is not present? Will we not get to this method?

Reply:
yeah, we should only get there if they have a pandas DataFrame (which would imply that pandas is available)